content/source/docs/enterprise/before-installing/reference-architecture/vmware.md

```diff
@@ -132,47 +88,41 @@ of this guide. You will be prompted for the public and private certificates duri
 
 ## Infrastructure Diagram
 
-![vmware-infrastructure-diagram](./assets/vmware-infrastructure-diagram.jpg)
+![vmware-mounted-disk-infrastructure-diagram](./assets/vmware-mounted-disk-infrastructure-diagram.jpg)
```

Comment:
In a markdown image tag, the text in the square brackets becomes the alt text for the image. Alt text should be a human-readable description of the image content, at about the same level of detail you'd expect a sighted reader to resolve when reading the image. (i.e., short descriptions for images a sighted reader would glance and nod at, and longer descriptions for mandatory images that a sighted reader would spend a minute perusing.)

Reply:
Thanks for the info @nfagerlund. I propose we leave this in the current commit and do a sweep of all the reference architecture content as I know it is all incorrectly using the alt text at the moment.